### PR TITLE
Seeding users with different roles

### PR DESCRIPTION
**Pull request by emcoding**
Students, coaches, organizers, a mentor and a supervisor
it "should be" DRY test FAILED

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1 +1,8 @@
+FactoryGirl.create_list(:student, 12)
+FactoryGirl.create_list(:coach, 3)
+FactoryGirl.create :mentor
+FactoryGirl.create :supervisor
```
Comment (yorickpeterse) [inline db/seeds.rb]: To keep things consistent with the other lines the usage of parenthesis here wouldn't be a bad thing.

**Comment by emcoding** [comment]
Is the formatting okay like it this? Can I close it?

File: db/seeds.rb [hunk 2]
```diff
@@ -1 +1,12 @@
+FactoryGirl.create_list(:student, 12)
+FactoryGirl.create_list(:coach, 3)
+FactoryGirl.create_list(:organizer, 2)
+FactoryGirl.create (:mentor)
+FactoryGirl.create (:supervisor)
```
Comment (yorickpeterse) [inline db/seeds.rb]: Whitespace before parenthesis isn't needed :P

**Comment by emcoding** [comment]
I wish my desk was as neat as this ;-)

**Comment by yorickpeterse** [comment]
Now it looks good to me :+1: :shipit:

**Comment by emcoding** [comment]
<3

**Pull request merged**